README.md

```diff
@@ -114,6 +116,103 @@ calculator to the constructor of all these object, a calculator is
 instantiated in the factory, and object instances are created through
```

Comment:
To alleviate the need to pass an instance of the
calculator to the constructor of all these **objects**, a calculator is
instantiated in the factory, and object instances are created through
the factory's **methods** `CreatePoint`, `CreateSegment`, etc.

Reply:
`methods` c'est deja correct, non?

Reply:
C'est corrige, sinon.

Reply:
Ben non c'est écrit `methodEs`... :-D

Reply:
Ahhh oue! j'etais focalise sur le `s`. C'est bon.